app/build.gradle

```diff
@@ -29,6 +29,9 @@ android {
         sourceCompatibility = JAVA_VERSION
         targetCompatibility = JAVA_VERSION
     }
+    kotlinOptions {
+        jvmTarget = JAVA_VERSION
```

Comment:
I don't believe we need any changes to `build.gradle` to use `ContextCompat.getSystemService()`, do we?

Reply:
That was for the `getSystemService` extension function, which inlines to `ContextCompat.getSystemService()`.

Reply:
Can we remove it?